AppSyncRealTimeClientTests/Support/RealtimeGatewayURLInterceptorTests.swift

```diff
@@ -26,5 +26,16 @@ class RealtimeGatewayURLInterceptorTests: XCTestCase {
             "URL should contain the realtime part"
         )
     }
+    
+    func testCustomDomainInterceptRequest() {
+        let url = URL(string: "https://api.example.com/graphql")!
+        let request = AppSyncConnectionRequest(url: url)
+        let changedRequest = realtimeGatewayURLInterceptor.interceptConnection(request, for: url)
+        XCTAssertEqual(changedRequest.url.scheme, "wss", "Scheme should be wss")
+        XCTAssertEqual(
+            changedRequest.url.absoluteString,
+            "wss://api.example.com/graphql/realtime",
```

Comment:
if unit test for custom checks for absoluteString, can you update standard unit test to check for absolute string as well? it would make it clear what's happening to the standard one, if anything